### DISCUSSION TITLE
关于更换jsdelivr CDN的问题

### DISCUSSION THREAD

**Discussion by skyone-wzw** [AlistGo/alist#559] [Q&A] [answered]
我用的是docker部署

不知道是不是我这个地区的问题，jsdelivr不是特别慢就是打不开

![image](https://user-images.githubusercontent.com/73385177/154668300-de738ff4-fedb-4efb-a152-b1d568195240.png)

于是我想把jsdelivr改成其他cdn或自己储存一份，该怎么改源代码中的链接呀

（我本来想进入容器看看，结果容器里只看到了一个程序）😅

感谢帮助🙏🙏🙏🙏🙏

**Comment by skyone-wzw** [comment, marked as answer]
经过一番寻找，找到了修改的地方😅抱歉打扰了🙏

**Comment by fengguowudi** [comment]
我建议jsdelivr可以更换前面的cdn为fastly，这个域名下的cdn比默认的快，使用的是fastly的cdn

**Reply by skyone-wzw** [reply]
只要是我完全不懂Go ~~（虽然似乎也不需要懂）~~， __也没找到指定CDN的方法__ ，然后翻了一下文档和 [alist-org/alist-web](https://github.com/alist-org/alist-web) 猜的这个项目的public目录就是静态文件目录，而CDN可以通过这一行这种方法设置 [build.sh](https://github.com/alist-org/alist-web/blob/v2/build.sh#L17)

感谢回复🙏

**Comment by simpleandstupid** [comment]
jsdelivr是多cdn负载均衡的，也提供fastly cdn（只是中国用户dns解析到的都是cloudflare cdn罢了）
（完全可以在dns上做手脚，让jsdelivr走fastly）

**Reply by fengguowudi** [reply]
这样太过麻烦，还不如指定域名
